src/pages/home/report/ReportActionItemMessage.js

```diff
@@ -84,7 +83,7 @@ function ReportActionItemMessage(props) {
     };
 
     return (
-        <View style={[styles.chatItemMessage, !props.displayAsGroup && isAttachment ? styles.mt2 : {}, ...props.style]}>
+        <View style={[styles.chatItemMessage, ...props.style]}>
```

Comment:
**confirming:** So, I see we no longer need this because we are handling it inside `ReportActionFragment`. But not sure exactly why this is being moved to that view. Is this the bug fix part?

Reply:
I think you meant:

> because we are handling it inside `AttachmentCommentFragment`

Yes, this is the fix part. This margin is explicitly meant for the attachment presentation (think: an actual displayed attachment image), but because of _various conditions_, when the message is deleted, the booleans here can be true, while deeper inside, we take a rendering path that doesn't actually display the attachment image. Hence the "jump".

In the new suggested structure, things are more clearly separated, so it's quite obvious that a margin meant for attachment can be displayed if-and-only-if we display an actual attachment image.